pkgs/applications/editors/emacs/build-support/generic.nix

```
@@ -26,7 +26,6 @@ in
 { pname
```

definition `version` is not used

---

Hey @inclyc, what about ignoring errors not relevant to the PR, such as existing ones?